frontend/package.json

```diff
@@ -67,10 +76,23 @@
     "start": "npm run develop",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
-    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1",
+    "test": "jest jest/config.js",
```

Comment:
Add bunch of commands to support running e2e tests in different settings as described in the README file

This also paves the path to run integration tests for each deployment stage later (will be implemented in a separate ticket)

Reply:
There are so many combination, I wouldn't know which one to actually use. Please add more description in the readme for each to say what each one does and why they are important.